gen-api-reference/README.md

```diff
@@ -41,37 +41,44 @@ at the root of the repo, and that your GOPATH is set properly. For example:
 
 # Generating the Pipelines API Reference Page
 
-These are instructions for generating a copy of the [Pipelines API Reference](https://www.kubeflow.org/docs/pipelines/reference/api/kubeflow-pipeline-api-spec/) page from the swagger file. 
+Use these instructions to generate an updated copy of the [Pipelines API Reference](https://www.kubeflow.org/docs/pipelines/reference/api/kubeflow-pipeline-api-spec/). 
 
 ## Prerequisites
-* Download the [pipelines](https://github.com/kubeflow/pipelines) repository.
+* Clone the [kubeflow/pipelines](https://github.com/kubeflow/pipelines) repository.
 * Install [npm](https://www.npmjs.com/get-npm).
 
 ## Installation
 
-1. Extract the pipelines repository to a local directory.
+1. From the kubeflow/pipelines repository, extract or download `pipelines\backend\api\swagger\kfp_api_single_file.swagger.json` to a local directory.
```

Comment:
Good point, since the instructions direct them to clone the repository I think that's a more natural way to do it. Removed the direction and put the following instruction into pre-reqs.